app/index.tsx

```
@@ -0,0 +1,21 @@
+import { Redirect, useRouter } from "expo-router";
+import { useAuthStore } from "@/store/useAuthStore";
+import { useEffect } from "react";
+
+export default function Index() {
+    const { hasOnboarded, token } = useAuthStore();
+
+    const router = useRouter();
+
+    useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on router.replace.

This dependency is being used here, but is not specified in the hook dependency list.

This dependency is being used here, but is not specified in the hook dependency list.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)